packages/permission-controller/src/rpc-methods/requestPermissions.test.ts

```diff
@@ -99,6 +99,7 @@ describe('requestPermissions RPC method', () => {
       delete expectedError.stack;
 
       const response: any = await engine.handle(req as any);
+      delete response.error.stack;
```

Comment:
the stack is in the error now... which was causing the tests to fail... not sure what caused this. But we delete it from the expected error (stacks don't match if you remove line 99).

Reply:
Hmm. Seems odd that we'd get this error now...

Reply:
seems like another version flattening is the culprit: https://github.com/MetaMask/core/pull/1064/files/27ab5415864d015794a0540f393a1d2eb5089e0e#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deL5138

not sure why this flattening is happening here. Gonna dig into it a bit.

Reply:
Ok I rebased/squashed and fixed these two issue that were arising from dedupes that were happening in the root `yarn.lock`. The diff in the lock files is much smaller and does not include any of these dedupes

Reply:
```delete response.error.stack;```

This line that was added is no longer added.

Reply:
Oh cool. How did you dedupe, out of curiosity?

Reply:
so the base lockfile is currently not deduped. I guess I had accidentally run yarn in the root of the directory which will automatically dedupe causing this and a few other issues. We should probably do that at some point. But it didn't need to be part of this PR.

Reply:
Ohh... that's really interesting. Good job looking into this. If I'm reading what you're saying right, it sounds like deduping is liable to cause unexpected issues? If so I feel like this is a really big gotcha.

Reply:
thats correct! You can test this by running `yarn` in the root directory and seeing that there is a diff generated. And then run tests and see that some are failing as a result

Reply:
Seems like something about the monorepo structure doesn't force this dedupe to occur when you add deps to sub packages within the repo.

Reply:
Gotcha. Thanks for the added info!